ci/LDKSwift/Sources/LDKSwift/batteries/ChannelManagerConstructor.swift

```diff
@@ -54,7 +55,7 @@ public class ChannelManagerConstructor: NativeTypeWrapper {
     private let chain_monitor: ChainMonitor
 
 
-    public init(channelManagerSerialized: [UInt8], channelMonitorsSerialized: [[UInt8]], keysInterface: KeysInterface, feeEstimator: FeeEstimator, chainMonitor: ChainMonitor, filter: Filter?, netGraphSerialized: [UInt8]?, txBroadcaster: BroadcasterInterface, logger: Logger, enableP2PGossip: Bool = false) throws {
+    public init(channelManagerSerialized: [UInt8], channelMonitorsSerialized: [[UInt8]], keysInterface: KeysInterface, feeEstimator: FeeEstimator, chainMonitor: ChainMonitor, filter: Filter?, netGraphSerialized: [UInt8]?, txBroadcaster: BroadcasterInterface, logger: Logger, enableP2PGossip: Bool = false, defaultConfig: UserConfig = UserConfig.initWithDefault(), payerRetries: Retry = Retry.initWithAttempts(a: UInt(3))) throws {
```

Comment:
is there a benefit to asking for a `Retry` object as opposed to just a number?

Reply:
Yes, it allows users to also specify a retry strategy based on `Retry::Timeout`, not only by number of attempts.

Reply:
cool. Looks good to me otherwise. Do you have a preference for using `defaultConfig` vs. `userConfig` for the public-facing argument name here?

Reply:
No, no strong opinion here, happy to change it.

Reply:
Alright, now renamed it in both `init` variants to `userConfig`.